revolv/static/js/base/accounting.js

```diff
@@ -0,0 +1,388 @@
+ * @param {String} project - The name of the project we are inserting under.
+ * @param {String} time - The time period in which this was inserted.
+ */
+function placeROIElement(data_to_insert, project, time) {
+	var total = data_to_insert['total'];
+	var revolv_earnings = data_to_insert['revolv_earnings'];
+	var retained_ROI = data_to_insert['retained_ROI'];
+
+	var className = '.project-' + cleanUpString(project);
+	$('.total-repayment').filter(className).append('<td class="added">' + String(total) + "</td>");
+	$('.revolv-earnings').filter(className).append('<td class="added">' + String(revolv_earnings) + "</td>");
+	$('.retained-ROI').filter(className).append('<td class="added">' + String(retained_ROI) + "</td>");
+}
+
+/**
+ * Places an AdminReinvestment element on the page.
+ * @param {Object} data_to_insert - A dictionary with values to insert,
+ * @param {String} project - The name of the project we are inserting under.
+ * @param {String} time - The time period in which this was inserted.
+ */
+function placeAdminInvestmentElement(data_to_insert, project, time) {
+	var total = data_to_insert['total'];
+
+	var className = '.project-' + cleanUpString(project);
+	$('.reinvestment').filter(className).append('<td class="added">' + String(total) + "</td>");
+
+}
+
+/**
+ * Places a cash in AdminAdjustment element on the page.
+ * @param {Object} data_to_insert - A dictionary with values to insert.
+ * @param {String} time - The time period in which this was inserted.
+ */
+var allCashInTransactions = {};
+function placeCashInAdjustmentElement(data_to_insert, time) {
+	var total = data_to_insert['total'];
+	var transactions = data_to_insert['transactions'];
+
+	var allTransactionsKeys = Object.keys(allCashInTransactions);
+	for (var i = 0; i < allTransactionsKeys.length; i++) {
+		allCashInTransactions[allTransactionsKeys[i]] = 0;
+	}
+	//insert all the transactions
+	var transactionKeys = Object.keys(transactions)
+	for (var i = 0; i < transactionKeys.length; i++) {
+		var transactionName = transactionKeys[i];
+		var transactionAmount = transactions[transactionName];
+		var cleanedTransactionName = cleanUpString(transactionName);
+		allCashInTransactions[cleanedTransactionName] = transactionAmount;
+	}
+	
+	allTransactionsKeys = Object.keys(allCashInTransactions);
+	for (var i = 0; i < allTransactionsKeys.length; i++) {
+		var name = allTransactionsKeys[i];
+		var amount = allCashInTransactions[name];
+		$('.transaction').filter('.' + name).append('<td class="added">' + amount + "</td>");
+	}
+
+	$('.cash-in-adjustment-total').append('<td class="added">' + total + "</td>");
+}
+
+/**
+ * Places a cash out AdminAdjustment element on the page.
+ * @param {Object} data_to_insert - A dictionary with values to insert.
+ * @param {String} time - The time period in which this was inserted.
+ */
+var allCashOutTransactions = {};
+function placeCashOutAdjustmentElement(data_to_insert, time) {
```

Comment:
Isn't this almost exactly the same as the last function? I.e. couldn't you just extract this functionality?